fenix/app/metrics.yaml

```diff
@@ -2985,10 +2985,8 @@ activation:
     lifetime: ping
     description: |
       A hashed and salted version of the Google Advertising ID from the device.
-      This will never be sent in a ping that also contains the client_id.
```

Comment:
Why is this being removed?

Reply:
Frank requested that we remove this string because we were sending it in a ping that contained the client_id. I can leave it since we are using a different metric, but I guess technically we are still sending the GaID in a ping with the client_id.